templates/contributors.html

```diff
@@ -26,7 +26,7 @@ <h1>Details of all the contributors</h1>
                 <p>teams:
                 {% for team in contributor.teams.all %}
                     {{ team.name }}
-                {% endfor %}
+                {% endfor %}{# for team in contributor.teams.all #}
```

Comment:
Shouldn't need to be necessary once you made the `check_end_label = False`.

Reply:
but gitmate is throwing error if I do not, do these changes inspite of ```0check_end_label = False```

Reply:
I think ```check_end_lables = False``` is not working and why it is not working , is it the problem of bear version maybe it release with new release

Reply:
okay thanks @Naveenaidu